src/forms/forms-list.component.tsx

```diff
@@ -60,7 +65,7 @@ export default function FormsList(props: FormsListProps) {
       getCurrentPatientUuid().subscribe(uuid => {
         getPatientEncounters(uuid).subscribe(encounters => {
           searchForms("POC").subscribe(forms => {
```

Comment:
Is the param insider searchForms() likely to change? If so, could we have a dynamic way of setting the param?

Reply:
I don't understand. Could you please elaborate?

Reply:
I am referring to the  `POC`value in searchForms.

Reply:
So, we didn't have ability to modify openmrs when we were starting out with poc, and could not modify openmrs to include form-attributes. So we adapted a naming-convention for our forms, in order to separate forms that can be rendered using angular form entry. So all forms that can be rendered using angular form entry, have 'POC' in them. What can we do about it now?